feature-tests/tests/resources/step-definitions/invoke-api-gateway-happy-path.step.ts

```diff
@@ -52,6 +52,86 @@ defineFeature(feature, (test) => {
     );
   });
 
+  test('Happy Path - Suspended Account - Get Request to /ais/userId - Returns Expected Data for <aisEventType>', ({
+    given,
+    when,
+    then,
+  }) => {
+    given(
+      /^I send an (.*) intervention message to the TxMA ingress SQS queue for a Account in (.*) state$/,
+      async (originalAaisEventType, aisEventType) => {
+        await sendSQSEvent(testUserId, originalAaisEventType);
+        await sendSQSEvent(testUserId, aisEventType);
+      },
+    );
+
+    when(
+      /^I invoke the API to retrieve the intervention status of the user's account. With history (.*)$/,
+      async (historyValue) => {
+        await timeDelayForTestEnvironment(1500);
+        response = await invokeGetAccountState(testUserId, historyValue);
+      },
+    );
+
+    then(
+      /^I expect the intervention to be (.*), with the following state settings (.*), (.*), (.*) and (.*)$/,
+      async (
+        interventionType: string,
+        blockedState: string,
+        suspendedState: string,
+        resetPassword: string,
+        reproveIdentity: string,
+      ) => {
+        console.log(`Received`, { response });
+        expect(response.intervention.description).toBe(interventionType);
+        expect(response.intervention.state.blocked).toBe(JSON.parse(blockedState));
+        expect(response.intervention.state.suspended).toBe(JSON.parse(suspendedState));
+        expect(response.intervention.state.resetPassword).toBe(JSON.parse(resetPassword));
+        expect(response.intervention.state.reproveIdentity).toBe(JSON.parse(reproveIdentity));
+      },
+    );
+  });
+
+  test('Happy Path - Blocked Account - Get Request to /ais/userId - Returns Expected Data for <aisEventType>', ({
+    given,
+    when,
+    then,
+  }) => {
+    given(
+      /^I send an (.*) intervention message to the TxMA ingress SQS queue for a Account in (.*) state$/,
+      async (originalAaisEventType, aisEventType) => {
+        await sendSQSEvent(testUserId, originalAaisEventType);
+        await sendSQSEvent(testUserId, aisEventType);
```

Comment:
as above, may be good to wait between sending the messages here.. just so we can be more confident they are processed in the order we need them to be

Reply:
![image](https://github.com/govuk-one-login/account-interventions-service/assets/110972905/1a28f938-d36e-45b4-9456-bc787af6fd32)
changes made and all tests still passing, thank you